packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -315,6 +315,31 @@ export function getValidatorApi({
       );
   }
 
+  async function getDefaultGraffiti(): Promise<string> {
+    const consensusCode = ClientCode.LS;
+    const consensusCommit = opts.version ?? "";
+
+    const lodestarClientVersion = {
+      code: consensusCode,
+      name: "Lodestar",
+      version: opts.version ?? "",
+      commit: consensusCommit,
+    };
+
+    const executionClientVersions = await chain.executionEngine
+      .getClientVersion(lodestarClientVersion)
+      .catch((_) => []);
```

Comment:
what if Lodestar is started before the EL, this request will likely fail and we will not have any execution client information